libs/dev_only_scripts/dump_form_data_to_sql_server.php

```diff
@@ -0,0 +1,180 @@
+    echo("Error description 4: " . $mysqli -> error) . "\n";
+    exit();
+}
+
+if (!$mysqli->query("CREATE TABLE indicators
+                        (
+                        indicatorID int(10) NOT NULL DEFAULT 0,
+                        name text  NOT NULL,
+                        format text  NOT NULL,
+                        description varchar(50)  DEFAULT NULL,
+                        default_data text  DEFAULT NULL,
+                        parentID int(10) DEFAULT NULL,
+                        categoryID varchar(20)  DEFAULT NULL,
+                        html text  DEFAULT NULL,
+                        htmlPrint text  DEFAULT NULL,
+                        jsSort varchar(255)  DEFAULT NULL,
+                        required int(10) NOT NULL DEFAULT 0,
+                        sort int(10) NOT NULL DEFAULT 1,
+                        timeAdded datetime NOT NULL,
+                        disabled int(10)  NOT NULL DEFAULT 0,
+                        is_sensitive int(10) NOT NULL DEFAULT 0
+                        )
+                        SELECT indicatorID,
+                        name,
+                        format,
+                        description,
+                        indicators.default as default_data,
+                        parentID,
+                        categoryID,
+                        html,
+                        htmlPrint,
+                        jsSort,
+                        required,
+                        sort,
+                        timeAdded,
+                        disabled,
+                        is_sensitive 
+                        FROM $portalToExport.indicators;")) {
+    echo("Error description 5: " . $mysqli -> error) . "\n";
+    exit();
+}
+
+if (!$mysqli->query("CREATE TABLE action_history
+                        (
+                        actionID int(10)  NOT NULL DEFAULT 0,
+                        recordID int(10)  NOT NULL,
+                        action_taken_by varchar(50)  NOT NULL,
+                        actionType varchar(50)  NOT NULL,
+                        action_time int(10)  NOT NULL,
+                        stepID int(10) DEFAULT 0,
+                        stepTitle varchar(64) 
+                        )
+                        SELECT 
+                        action_history.actionID,
+                        action_history.recordID,
+                        action_history.userID as action_taken_by,
+                        action_history.actionType,
+                        action_history.time as action_time,
+                        workflow_steps.stepID,
+                        workflow_steps.stepTitle
+                        FROM $portalToExport.action_history
+                        LEFT JOIN $portalToExport.workflow_steps on workflow_steps.stepID = action_history.stepID;")) {
+    echo("Error description 6: " . $mysqli -> error) . "\n";
+    exit();
+}
+$filename = $tempDBName . ".sql";
+$directory = "/var/www/html/sqlserver_dumps/";

```

Comment:
Would this expose the files of the database export over /html/ if the correct path was guessed?